test/test_testmon.py

```diff
@@ -192,6 +192,25 @@ def test_add():
         assert tuple(res) == (1, 0, 0), res
         sys.modules.pop('test_a', None)
 
+    def test_tlf(self, testdir):
+        tf = testdir.makepyfile(test_a="""
+            def test_add():
+                1/0
+        """, )
+        testdir.inline_run("--testmon", "-v")
+        sys.modules.pop('test_a', None)
+
+        result = testdir.runpytest("--testmon", "-v")
+        result.stdout.fnmatch_lines([
+            "*1 failed, 1 deselected*",
+        ])
+        tf.setmtime(1424880936)
```

Comment:
Could benefit from a comment that this is in the past (`datetime.datetime(2015, 2, 25, 17, 15, 36)`).